### ISSUE TITLE
Conflict between Select highlighted and Zooming

### ISSUE THREAD

**Issue by Xan-Kun** [danvk/dygraphs#569]
If highlighting and seleting (locking) a series in enabled and you end a zoom selection on a series, this series will get highlighted forever.
Even after (regular) selecing a new one and deselcting "all", the previously and accidentally selected series still stays highlighted.
It happens in all examples on the gallery.

I will post a workaround if I find one :-)

**Comment by danvk** [comment]
I'm not sure what you mean. Are you referring to a specific gallery example? What is locking?

**Comment by Xan-Kun** [comment]
Thanks for the answer and sorry for my delay. It is a problem I am working on for days now, and not seem able to find a suitable fix.

In 
http://dygraphs.com/tests/series-highlight.html
if I zoom in, one of the series gets drawn thicker then the others, but still transparent.
The effect is more significant in the first two examples.
I thin it keeps the line thicknes of the series that was closest to the curser when the zoom selection is finished (mouseup).
Sorry for the struggling explanation, maybe a screenshot will tell more.
Basically I zoomed in and out again (with doubleclick).
![zoomhighlightartefact](https://cloud.githubusercontent.com/assets/11686614/7401987/82742ad2-eec8-11e4-8b44-04f7ab96ac30.png)

I tried Firefox, Chrome and Opera.

**Comment by Xan-Kun** [comment]
P.S.: The strange thing is, it is neither "corrected" by
clearSelection();
nor
setSelection(false);

**Comment by Xan-Kun** [comment]
I think when selecting a zoom area, right after stopping the mous but before releasing the mousebutton, a series is highlighted and gets a bigger strokeWidth. When releasing the button the actual zooming is done, but the previous strokeWidth for this series stays the same.
